src/OmniSharp/Api/v1/Highlighting/OmnisharpController.Highlighting.cs

```diff
@@ -48,7 +48,7 @@ public partial class OmnisharpController
                 }
                 else
                 {
-                    foreach (var line in request.Lines)
+                    foreach (var line in request.Lines.Where(z => z <= text.Lines.Count))
```

Comment:
The document should be updated with the content from the request before it gets to the controller, so I'm not sure how it could ever be different.